source/_integrations/legrand_homepluscontrol.markdown

```diff
@@ -0,0 +1,103 @@
+---
+title: Legrand Home+ Control
+description: Instructions on how to integrate Legrand Home+ Control into Home Assistant.
+ha_category:
+  - Switch
+ha_release: '2021.03'
+ha_iot_class: Cloud Polling
+ha_codeowners:
+  - '@chemaaa'
+ha_config_flow: true
+ha_domain: homepluscontrol
+---
+
+The `Home+ Control` integration platform allows you to control a range of Legrand in-wall switches and power outlets that have smarthome functionality thanks to their "with Netatmo" capabilities. 
+
+This integration works against the `Home+ Control` API, which is one of the many APIs offered through the *Works with Legrand* program (https://developer.legrand.com/). The API is capable of managing "Legrand/Btcino with Netatmo" devices, such as light switches, power outlets and rolling shutters.
+
+The devices that this API can manage are offered in different designs across different countries. The details of these can be found in the following page: https://developer.legrand.com/solutions/wiring-devices-with-netatmo/
```

Comment:
It is better to use text and link that (avoid bare URLs).

However, if needed, make sure at least it is wrapped:

```suggestion
The devices that this API can manage are offered in different designs across different countries. The details of these can be found on the following page: <https://developer.legrand.com/solutions/wiring-devices-with-netatmo/>
```

Reply:
Noted. I have now modified this to use text with a link.